_sass/blocks/_home-explore.scss

```diff
@@ -0,0 +1,47 @@
+// Explore data callout on homepage
+//
+// Styleguide blocks.explore-home
+
+.explore_home-heading {
+  margin-bottom: 0;
+}
+
+.explore_home-subhead {
+  @include heading(para-md);
+  font-weight: $weight-bold;
+  margin-bottom: 0;
+}
+
+.explore_home-tabs {
+  @include heading(para-sm);
+  margin-bottom: 2px;
+  text-align: right;
+
+  a {
+    color: $black;
+    margin-right: $base-padding-lite;
+    padding: $base-padding-lite $base-padding $base-padding-lite $base-padding;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `$base-padding-lite $base-padding` instead of `$base-padding-lite $base-padding $base-padding-lite $base-padding`